src/components/js/EcInstalledAppCard.js

```diff
@@ -1,5 +1,6 @@
 import { i18n, formatDate } from '@ecomplus/utils'
 import EcomApps from '@ecomplus/apps-manager'
+import { FadeTransition } from 'vue2-transitions'
```

Comment:
```suggestion
import { FadeTransition } from 'vue2-transitions'
```

Não estamos usando fade neste componente, não estava antes nem agora com sua implementeção..